pkg/webui/console/views/device/index.js

```diff
@@ -151,6 +165,12 @@ export default class Device extends React.Component {
         link: payloadFormattersLink,
         exact: false,
       },
+      {
+        title: sharedMessages.claiming,
+        name: 'claim-auth-code',
+        link: `${basePath}/claim-auth-code`,
+        disabled: !hasJs,
```

Comment:
(I'd actually prefer to add a `mayViewOrSetClaimAuthenticationCode` feature check for this. Though the feature checks are not configured to include external conditions (`supports_join`), so it doesn't seem as straightforward. Okay to leave it like this then, unless you have another idea here.)

Reply:
From the PR:
> I will also try to find an appropriate abstraction for checking whether the end device has cluster js/as/ns addresses rather than just comparing those in the render function. Will add this in a separate PR with disabling the payload formatters tab when as is disabled or not in cluster.